ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/util/BatchWrite.java

```diff
@@ -86,6 +88,8 @@ private static Distribution exponentialDistribution(double base, double expStart
   private final long maxBytes;
   private final int maxMessages;
   private final Duration maxDelay;
+  private final String shortClassName = this.getClass().getName().replaceAll(".*[.]", "");
+  private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
Or is the point that we want to catch subclass names?

For a logger, I'm used to the logger being static and the log messages refer to the superclass where it was logged rather than the concrete subclass. I think I expect the same for metrics, but I'm open to thoughts there.

Reply:
this was actually leftover from the previous method. I removed this change.